### PR TITLE
Feature/datastore create

### LINKED ISSUE #1: all unit test cases are not being executed via tox
There appears to be a total of 106 unique unit test cases within the ./tests/unit directory structure

`python -m unittest discover` executes the 34 test cases directory under ./tests/unit, but skips the 63 tests that are under ./tests/unit/resources.  

out of the 63 tests that are under ./tests/unit/resources, 17 currently fail

The solution is to create an empty file called init.py to ./tests/unit/resources directory in order to allow the unittest discover routine to find the test cases

### PR DESCRIPTION
**Pull request by svt-kmcdaniel**
### Description
Add support for the datastore post endpoint that will enable the consumer to create SimpliVity datastores

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Comment by svt-kmcdaniel** [comment]
@sijeesh can you please let me know if the modified CHANGELOG.md that is present in this Pull Request is up to par with the expectations of this project?

**Review by svt-mpilon — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,51 @@
...
+    - endpoint support for /policies <GET>
+    - endpoint support for /virutal_machines <GET>
+    - endpoint support for /virutal_machines/set_policy <POST>
+    - endpoint support for /virutal_machines/{vmId} <GET>
+    - endpoint support for /virutal_machines/{vmId}/backup <POST>
+    - endpoint support for /virutal_machines/{vmId}/backup_parameters <POST>
+    - endpoint support for /virutal_machines/{vmId}/backups <GET>
+    - endpoint support for /virutal_machines/{vmId}/clone <POST>
+    - endpoint support for /virutal_machines/{vmId}/move <POST>
+    - endpoint support for /virutal_machines/{vmId}/set_policy <POST>
```
Comment (sijeesh) [inline CHANGELOG.md:51]: Nicely formatted, I think we can go ahead with this format.
Make sure we are updating the endpoint support file as well when we are adding new endpoints support.

Comment (sijeesh) [reply CHANGELOG.md:51]: Let's update this ChangeLog(Under an unreleased version title) whenever we make an update/ bug fix.

File: simplivity/resources/datastores.py [hunk 2]
```diff
@@ -105,6 +107,41 @@ def get_by_data(self, data):
         """
         return Datastore(self._connection, self._client, data)
 
+    def create(self, datastore_name, cluster, policy="Fixed Default Backup Policy", size=0, timeout=-1):
```
Comment (sijeesh) [inline simplivity/resources/datastores.py]: Do we need to hardcode the default policy name here? 
name of the default policy will be the same for all the OVC versions?
is policy a mandatory field?

**Review by sijeesh — COMMENTED** [review]
Thank you for spending time to clean the SDK code and documentation. 
Looks good to me, please have a look at my questions.

File: simplivity/resources/datastores.py (hunk 2 above)
Comment (svt-kmcdaniel) [reply simplivity/resources/datastores.py]: A policy is required when creating a SimpliVity datastore.

I will remove the "Fixed Default Backup Policy" here and adjust the example scripts.

All releases have the "Fixed Default Backup Policy" but talking to the engineering team, the intent is not to assign it to a datastore, but to indivdual VMs

**Comment by svt-kmcdaniel** [comment]
I believe this PR now ready to be merged

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**